property-services/src/main/java/org/egov/pt/repository/PropertyRepository.java

```diff
@@ -63,28 +64,33 @@
 	public List<Property> getProperties(PropertyCriteria criteria, Boolean isApiOpen) {
 
 		List<Object> preparedStmtList = new ArrayList<>();
-		String query = queryBuilder.getPropertySearchQuery(criteria, preparedStmtList);
+		Boolean isPlainSearch = true;
+		String query = queryBuilder.getPropertySearchQuery(criteria, preparedStmtList, isPlainSearch);
 		if (isApiOpen)
 			return jdbcTemplate.query(query, preparedStmtList.toArray(), openRowMapper);
 		else
 			return jdbcTemplate.query(query, preparedStmtList.toArray(), rowMapper);
 	}
 
+
 	public List<Property> getPropertiesForBulkSearch(PropertyCriteria criteria) {
 		List<Object> preparedStmtList = new ArrayList<>();
-		String query = queryBuilder.getPropertyQueryForBulkSearch(criteria, preparedStmtList);
+		Boolean isPlainSearch = true;
+		String query = queryBuilder.getPropertyQueryForBulkSearch(criteria, preparedStmtList, isPlainSearch);
 		return jdbcTemplate.query(query, preparedStmtList.toArray(), rowMapper);
 	}
 
 	public List<String> fetchIds(PropertyCriteria criteria) {
 		List<Object> preparedStmtList = new ArrayList<>();
 		String basequery = "select id from eg_pt_property";
 		StringBuilder builder = new StringBuilder(basequery);
-		if (!ObjectUtils.isEmpty(criteria.getTenantId())) {
-			builder.append(" where tenantid=?");
-			preparedStmtList.add(criteria.getTenantId());
+		if (!ObjectUtils.isEmpty(criteria.getTenantIds()) && !ObjectUtils.isEmpty(criteria.getFromDate()) && !ObjectUtils.isEmpty(criteria.getToDate())) {
+			builder.append(" where tenantid IN (:tenantIds) BETWEEN fromDate=? AND toDate=? ");
+			preparedStmtList.add(criteria.getTenantIds());
+			preparedStmtList.add(criteria.getFromDate());
+			preparedStmtList.add(criteria.getToDate());
 		}
-		String orderbyClause = " order by lastmodifiedtime,id offset ? limit ?";
+		String orderbyClause = " order by lastmodifiedtime,id offset ? limit ? ";
```

Comment:
don't use id.  use propertyid.

Reply:
methods fetchIds and getPropertyId were there from the start. I haven't added those methods